src/settingsmanager.cpp

```diff
@@ -81,26 +81,20 @@ bool SettingsManager::setDownloadDir(QString downloadDir)
     return true;
 }
 
-bool SettingsManager::setMonitorDir(QString monitorDir)
+bool SettingsManager::setMonitorDir(QString monitorDir, QStringList zimList)
 {
     m_monitorDir = monitorDir;
     m_settings.setValue("monitor/dir", monitorDir);
+    m_monitorDirZims = zimList;
```

Comment:
`SettingsManager` doesn't seem the right place for keeping track of the ZIM files in the monitor dir. I think it should belong to `Library`.